commit bb09a1d7e2c5dd29bbdfc70a24bd35ab5a0ce60a
Author: Ronen Friedman <rfriedma@redhat.com>
Date:   Sun Oct 17 06:21:16 2021 +0000

    common: hide internal logger configuration strings from clients
    
    Replace the common pattern used by clients of the logger, of:
    - creating multiple "one element maps" for all config options;
    - passing them, untouched, to the logger's update config.
    with one that hides the configuration handling details from
    the users.
    
    Note: a followup PR, bypassing the strmaps construction altogether,
    should be considered.
    
    Signed-off-by: Ronen Friedman <rfriedma@redhat.com>

diff --git a/src/mgr/MgrStandby.cc b/src/mgr/MgrStandby.cc
index f3d1b38770a..96c18892d0a 100644
--- a/src/mgr/MgrStandby.cc
+++ b/src/mgr/MgrStandby.cc
@@ -375,29 +375,8 @@ void MgrStandby::respawn()
 
 void MgrStandby::_update_log_config()
 {
-  map<string,string> log_to_monitors;
-  map<string,string> log_to_syslog;
-  map<string,string> log_channel;
-  map<string,string> log_prio;
-  map<string,string> log_to_graylog;
-  map<string,string> log_to_graylog_host;
-  map<string,string> log_to_graylog_port;
-  uuid_d fsid;
-  string host;
-
-  if (parse_log_client_options(cct, log_to_monitors, log_to_syslog,
-			       log_channel, log_prio, log_to_graylog,
-			       log_to_graylog_host, log_to_graylog_port,
-			       fsid, host) == 0) {
-    clog->update_config(log_to_monitors, log_to_syslog,
-			log_channel, log_prio, log_to_graylog,
-			log_to_graylog_host, log_to_graylog_port,
-			fsid, host);
-    audit_clog->update_config(log_to_monitors, log_to_syslog,
-			      log_channel, log_prio, log_to_graylog,
-			      log_to_graylog_host, log_to_graylog_port,
-			      fsid, host);
-  }
+  clog->parse_client_options(cct);
+  audit_clog->parse_client_options(cct);
 }
 
 void MgrStandby::handle_mgr_map(ref_t<MMgrMap> mmap)
